assets/src/edit-story/app/uploader/index.js

```diff
@@ -0,0 +1 @@
+export { default as useUploader } from './useUploader';
```

Comment:
Probably should add the Copyright / license text to the beginning of the files.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/195/commits/2524ec2a509e318fb844aea85296c21ca558e664